tests/pipeline/resources/no-topics/config.yaml

```diff
@@ -0,0 +1,5 @@
+environment: development
```

Comment:
aren't those the same contents as the default `config.yaml`?

Reply:
indeed they are.. i can get rid of one of them